desktop/extensions-sdk/guides/use-docker-socket-from-backend.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
I created this page from the paragraph that was in the metadata.md initially. I didn't change anything here. I was thinking that it could be improved in another PR.